Unable to build on macOS 13.0 Ventura

Attempting to use Homebrew to install, or manually build from source, results in this error:

```bash
🕙 15:24:37 jeffbyrnes in rsgain/build on  master
♠ cmake .. -DCMAKE_BUILD_TYPE=Release
-- The CXX compiler identification is AppleClang 14.0.0.14000029
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /Library/Developer/CommandLineTools/usr/bin/c++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Found PkgConfig: /usr/local/bin/pkg-config (found version "0.29.2")
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD - Success
-- Found Threads: TRUE
-- Checking for module 'libavcodec'
--   Found libavcodec, version 59.37.100
-- Checking for module 'libavformat'
--   Found libavformat, version 59.27.100
-- Checking for module 'libswresample'
--   Found libswresample, version 4.7.100
-- Checking for module 'libavutil'
--   Found libavutil, version 57.28.100
-- Checking for module 'taglib>=1.12'
--   Found taglib, version 1.13
-- Checking for module 'libebur128>=1.2.4'
--   Found libebur128, version 1.2.6
-- Checking for module 'inih'
--   Found inih, version 56
-- Checking for module 'fmt'
--   Found fmt, version 9.1.0
-- Configuring done
-- Generating done
-- Build files have been written to: /Users/jeffbyrnes/dev/rsgain/build

🕙 15:24:47 jeffbyrnes in rsgain/build on  master [?]
♠ make
[ 16%] Building CXX object src/CMakeFiles/rsgain.dir/rsgain.cpp.o
In file included from /Users/jeffbyrnes/dev/rsgain/src/rsgain.cpp:54:
/Users/jeffbyrnes/dev/rsgain/src/output.hpp:90:24: error: field has incomplete type 'struct winsize'
        struct winsize ws;
                       ^
/Users/jeffbyrnes/dev/rsgain/src/output.hpp:90:16: note: forward declaration of 'winsize'
        struct winsize ws;
               ^
1 error generated.
make[2]: *** [src/CMakeFiles/rsgain.dir/rsgain.cpp.o] Error 1
make[1]: *** [src/CMakeFiles/rsgain.dir/all] Error 2
make: *** [all] Error 2
```

Opening `output.hpp` in Visual Studio Code triggers showing the same issue on that line.

--- a/config/config.h.in
+++ b/config/config.h.in
@@ -3,6 +3,6 @@
 #define PROJECT_VERSION "@PROJECT_VERSION@"
 #define PROJECT_URL "@PROJECT_HOMEPAGE_URL@"
 
-#ifdef __unix__
+#ifndef _WIN32
 #define PRESETS_DIR "@PRESETS_DIR@"
 #endif

--- a/src/easymode.cpp
+++ b/src/easymode.cpp
@@ -455,7 +455,7 @@ static void load_preset(const char *preset)
     // Find preset file from name
     if (!path.has_extension()) {
 
-    // Mac/Windows check user directory before system directory
+        // Mac/Linux check user directory before system directory
 #ifndef _WIN32
 #ifdef __APPLE__
         join_paths(path, {(const char*) getenv("HOME"), "Library", EXECUTABLE_TITLE, "presets", preset});

--- a/src/easymode.cpp
+++ b/src/easymode.cpp
@@ -455,8 +455,8 @@ static void load_preset(const char *preset)
     // Find preset file from name
     if (!path.has_extension()) {
 
-        // Mac/Linux check user directory before system directory
-#ifdef __unix__
+    // Mac/Windows check user directory before system directory
+#ifndef _WIN32
 #ifdef __APPLE__
         join_paths(path, {(const char*) getenv("HOME"), "Library", EXECUTABLE_TITLE, "presets", preset});
 #else

--- a/src/output.hpp
+++ b/src/output.hpp
@@ -39,6 +39,10 @@
 #include <sys/ioctl.h>
 #endif
 
+#ifdef __APPLE__
+#include <util.h>
+#endif
+
 #ifdef _WIN32
 #include <windows.h>
 #endif

